cpu/cpu_freebsd.go

```diff
@@ -103,7 +103,9 @@ func InfoWithContext(ctx context.Context) ([]InfoStat, error) {
 	if u32, err = unix.SysctlUint32("hw.clockrate"); err != nil {
 		return nil, err
 	}
-	c.Mhz = float64(u32)
+	c.Mhz.Current = float64(u32)
+	c.Mhz.Min = 0
+	c.Mhz.Max = 0
```

Comment:
Please consider writing this


```suggestion
	c.Mhz = Mhz{Current: float64(u32)}
```

Abd everywhere you did the replacement